torchao/_executorch_ops.py

```diff
@@ -3,7 +3,207 @@
 #
 # This source code is licensed under the BSD 3-Clause license found in the
 # LICENSE file in the root directory of this source tree.
+
+from typing import Tuple
+
 import torch
+from torch.library import Library, impl
+
+quantized_decomposed_lib = Library("quantized_decomposed", "FRAGMENT")
+
+
+# ============================
+# |  quantize_per_token_aqt  |
+# ============================
+
+
+quantized_decomposed_lib.define(
+    "quantize_per_token_aqt(Tensor input, Tensor scales, Tensor zero_points, "
+    "int quant_min, int quant_max, ScalarType dtype) -> Tensor"
+)
+
+
+@impl(quantized_decomposed_lib, "quantize_per_token_aqt", "CompositeExplicitAutograd")
```

Comment:
why adding new ops instead of reusing the quantize_affine and dequantize_affine ops?

Reply:
It's because executorch expects these custom ops when lowering the quantized model. Seems like making `quantize_affine` a custom op is a bit overkill

Reply:
quantize_affine/dequantize_affine are already preserved as custom ops and supported by xnnpack lowering though

Reply:
see https://github.com/pytorch/ao/blob/d32afefdb6b8420cc31c9ed307497f4d305dd49c/test/integration/test_integration.py#L2039-L2041, does this work?

Reply:
oh I see, yeah let me give that a try then. Does that mean executorch and XNNPACK need to understand all the arg types, like `MappingType` and `ZeroPointDomain`?

Reply:
these args are converted to strings, yeah they'll need to understand the strings

Reply:
Ok updated, take another look?